.github/CODEOWNERS

```diff
@@ -29,5 +29,8 @@
 ###########
 /eng/   @weshaggard @mitchdenny @danieljurek
 /eng/code-quality-reports/ @mssfang @JonathanGiles
+/eng/jacoco-test-coverage/ @mssfang @JonathanGiles
+/eng/spotbugs-aggregate-report/ @mssfang @JonathanGiles
```

Comment:
Change @mssfang to @srnagar for both of these, as he set them up. I'm happy to be co-codeowner.